content/en/continuous_testing/explorer/search_runs.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Search Test Runs
+kind: documentation
+description: Examine all of your test runs and troubleshoot failing test results.
+further_reading:
+- link: '/continuous_testing/explorer'
+  tag: 'Documentation'
+  text: 'Learn about the Synthetic Monitoring & Continuous Testing Explorer'
+---
+
+## Overview
+
+After selecting a time frame from the dropdown menu on the top right, you can search for test runs by clicking on the **Test Runs** event type in the [Synthetic Monitoring & Continuous Testing Explorer][1].
+
+You can use facets to accomplish the following actions:
+
+- Observe the latest test runs that required retries.
+- Aggregate failed API test runs by HTTP status code and plot trends. 
+
+## Explore facets
+
+The facets panel on the left lists several facets you can use to search through your test runs. To start customizing the search query, click through the list of facets starting with **Common**.
+
+### Common test run attributes
+
+**Commmon** facets allow you to filter on your test runs' attributes.
+
+| Facet            | Description                                                                                             |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| `Batch ID`        | The batch ID associated with the test run.                                               |
+| <code>Execution&nbsp;Rule</code> | The execution rule associated with the test result of the CI batch: `Blocking`, `Non Blocking`, and `Skipped`. |
+| `Location`       | The location associated with the test result of the batch.                                              |
+| `Passed`        | The overall status of the test run.                                               |
+| `Run Type`      | The run type of the test run. It can be scheduled, CI, or manually triggered.                                             |
+
+### Timings attributes
+
+**Timings** facets allow you to filter on timing related attributes.
+
+| Facet          | Description                                 |
+|----------------|---------------------------------------------|
+| `DNS`  | The CI provider associated with the batch.  |
```

Comment:
We need to update these descriptions, they slipped through my draft sorry about that! Let's go with the following: 

```
DNS: The time spent resolving the DNS name for an API test run
Download: The time spent downloading the response for an API test run
First Byte: The time spent waiting for the first byte of the response to be received for an API test run
Open: The overall time a websocket remained open for a WebSocket test run
Received: The overall time a websocket connection spent receiving data for a WebSocket test run
TCP: The time spend establishing a TCP connection for an API test run
Total: The total response time for an API test run
```